hack/build-rpms.sh

```diff
@@ -22,7 +21,7 @@ os::build::setup_env
 
 if [[ "${OS_ONLY_BUILD_PLATFORMS:-}" == 'linux/amd64' ]]; then
 	# when the user is asking for only Linux binaries, we will
-	# furthermore not build cross-platform clients in tito
```

Comment:
am i right in thinking that currently tito is no longer in used (e.g it was in 3.x but world moved on since then? )

Reply:
I will need some clarification on where it is still used today, if at all.

Reply:
I’m not aware of any use in this repo, and it hasn’t been updated in a year or more.